lib/roar/json/json_api/nesting.rb

```diff
@@ -0,0 +1,35 @@
+module Roar
+  module JSON
+    module JSONAPI
+      module Nesting
+        def nested(name, options = {}, &block)
+          options = options.merge(
+            getter:   ->(_opts) { self },
+            setter:   ->(opts)  {},
+            instance: ->(_opts) { self }
+          )
+
+          if block
+            options[:_nested_builder] = nested_builder
```

Comment:
Reimplementing the whole method in order to override `options[:_nested_builder]`:
https://github.com/trailblazer/representable/blob/master/lib/representable/declarative.rb#L30